### ISSUE TITLE
Java 9

### ISSUE THREAD

**Issue by fabianbergmark** [flyway/flyway#1904]
###### What version of Flyway are you using?

5.0.6

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin, SBT plugin, ANT tasks)

Java API in a Java 9 module

###### What database are you using (type & version)?

sqlite3

###### What operating system are you using?

Linux with Java 9

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, and the command you ran)

Flyway.migrate

###### What did you expect to see?

Flyway running my migrations

###### What did you see instead?

Exception in thread "main" org.flywaydb.core.api.FlywayException: Unable to obtain inputstream for resource: org/flywaydb/core/internal/version.txt
       at ...@1.0/org.flywaydb.core.internal.util.scanner.classpath.ClassPathResource.loadAsString(Unknown Source)
       at ...@1.0/org.flywaydb.core.internal.util.VersionPrinter.printVersion(Unknown Source)
       at ...@1.0/org.flywaydb.core.Flyway.execute(Unknown Source)

and

Exception in thread "main" org.flywaydb.core.api.FlywayException: Unable to obtain inputstream for resource: org/flywaydb/core/internal/database/sqlite/createMetaDataTable.sql
       at ...@1.0/org.flywaydb.core.internal.util.scanner.classpath.ClassPathResource.loadAsString(Unknown Source)
       at ...@1.0/org.flywaydb.core.internal.database.Database.getRawCreateScript(Unknown Source)
       at ...@1.0/org.flywaydb.core.internal.schemahistory.JdbcTableSchemaHistory.create(Unknown Source)
       at ...@1.0/org.flywaydb.core.internal.command.DbMigrate.migrate(Unknown Source)
       at ...@1.0/org.flywaydb.core.Flyway$1.execute(Unknown Source)
       at ...@1.0/org.flywaydb.core.Flyway$1.execute(Unknown Source)
       at ...@1.0/org.flywaydb.core.Flyway.execute(Unknown Source)

**Comment by axelfontaine** [comment]
I am unable to reproduce this. Please share a small test repo that reproduces the issue.

**Comment by axelfontaine** [comment]
Any news?

**Comment by axelfontaine** [comment]
Closing due to lack of response.

**Comment by pupeno** [comment]
I am experiencing this issue. It started when I upgraded from Java 8 to 10 and added a module-info.java to the project. It works fine when the application starts, but not from the tests.

Also potentially related, when upgrading to Java 10, getting resources by using a class stopped working and I needed to use a different one so that the class loader would match. More details about it here: https://stackoverflow.com/questions/49702287/getresourceasstream-returning-null-in-java-10
